index.html

```diff
@@ -1,76 +1,290 @@
+
+            <li class="back-end">BACK END DEV</li>
+            <li class="year">2015</li>
+          </ul>
+        </nav>
+        <p class="text">
+          A daily selection of private personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li class="language-type">HTML</li>
+          <li class="language-type">CSS</li>
+          <li class="language-type">JavaScript</li>
+        </ul>
+        <button class="project-button">See project</button>
+      </div>
+    </section>
+    <section class="second-phase">
+      <div class="second-page">
+        <img
+          src="images/2nd-phase/art.png"
+          alt="image"
+          height="220px"
+          width="295px"
+        />
+
+        <div class="bar-word">Tonic</div>
+        <nav class="nav-list">
+          <ul class="nav-items">
+            <li class="canopy">CANOPY</li>
+
+            <li class="back-end">BACK END DEV</li>
+            <li class="year">2015</li>
+          </ul>
+        </nav>
+        <p class="text">
+          A daily selection of private personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li class="language-type">HTML</li>
+          <li class="language-type">CSS</li>
+          <li class="language-type">JavaScript</li>
+        </ul>
+        <button class="project-button">See project</button>
+      </div>
+    </section>
+
+    <div class="about">
+      <h1 class="about-bigger-text">About Myself</h1>
+      <p class="about-text">
+        Hello I'am a software developer! I can help you build a product, feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don't hesitate to
+        contact me.
+      </p>
+    </div>
+    <h3 class="connect">LET'S CONNECT</h3>
+
+    <ul class="social">
+      <li>
+        <a href="#" class="icons-link">
+          <img src="images/icon_image/github.png" alt="Github" class="github"
+        /></a>
+      </li>
+      <li class="icons-li">
+        <a href="#" class="icons-link">
+          <img
+            src="images/icon_image/linkedin.png"
+            alt="linkedin"
+            class="linkedin"
+        /></a>
+      </li>
+      <li class="icons-li">
+        <a href="#" class="icons-link">
+          <img src="images/icon_image/angelist.png" alt="Hand" class="angelist"
+        /></a>
+      </li>
+      <li class="icons-li">
+        <a href="#" class="icons-link">
+          <img
+            src="images/icon_image/twitter.png"
+            alt="twitter"
+            class="twitter"
+        /></a>
+      </li>
+      <li class="icons-li">
+        <a href="#" class="icons-link">
+          <img src="images/icon_image/medium.png" alt="Hand" class="medium"
+        /></a>
+      </li>
+    </ul>
+
+    <button class="resume-button">Get my resume</button>
+
+    <ul class="about-list">
```

Comment:
- [ ] Please I highly suggest aligning the content of `about-list` class with CSS flexbox properties to implement a better user experience on the devices like tablets and desktops. 😄